Cloud changelog - Jan 26th (#6151)
mattermost/docs: source/install/cloud-changelog.md

Latest Mattermost Cloud releases:

- [Release 2023-01-19](#release-2023-01-19)
- [Release 2023-01-12](#release-2023-01-12)
- [Release 2022-12-20](#release-2022-12-20)
- [Release 2022-12-01](#release-2022-12-01)
- [Release 2022-11-24](#release-2022-11-24)
- [Release 2022-11-17](#release-2022-11-17)

## Release 2023-01-19

### Improvements

#### User Interface (UI)

>  - Insights and drafts are now included when navigating through channels on the left-hand side using ALT+up/down arrows.

Reviewer: ```suggestion
 - Insights and drafts are now included when navigating through channels in the channel sidebar using ALT+UP/DOWN arrow keyboard keys.
```

#### Administration

>  - The max size of uploaded emojis is now reduced to 512KiB to reduce image download bandwidth.

Reviewer: ```suggestion
 - The maximum file size of uploaded emojis is now reduced to 512KB to reduce image download bandwidth.
```

Reviewer: Has this new file size limit been updated in-product as well?

Reviewer 2: Yes

 - Users can now monitor the progress of the bulk export job via its metadata field. It is available at ``mmctl export job show <jobID>``.
 - Compliance exports no longer timeout when uploading to S3.

>  - Users can now supply a CA file and client certificates for the Elasticsearch client.

Reviewer: Is the term "CA file" clear to the target audience? (I ask because I'm not 100% sure what a CA file is)

 - Grafana metrics are now available for database connection metrics. They are:
    - ``max_open_connections``
    - ``open_connections``
    - ``in_use_connections``
    - ``idle_connections``
    - ``wait_count_total``
    - ``wait_duration_seconds_total``
    - ``max_idle_closed_total``
    - ``max_idle_time_closed_total``
    - ``max_lifetime_closed_total``

### Bug Fixes
 - Fixed an issue with the plugin ``/public`` handling for subpaths.

>  - Fixed an issue where clicking **Pinned** on a post in the Threads view would result in the right-hand side being stuck in a loading state.

Reviewer: ```suggestion
 - Fixed an issue where selecting **Pinned** on a post in the Threads view would result in the right-hand side being stuck in a loading state.
```

#### User Interface (UI)
 - Insights and drafts are now included when navigating through channels in the channel sidebar using ALT+UP/DOWN arrow keyboard keys.
 - Added an onboarding tour point for Global Drafts.

#### Administration
 - The maximum size of uploaded emojis is reduced to 512KB to reduce image download bandwidth.
 - Users can now monitor the progress of the bulk export job via its metadata field. It is available at ``mmctl export job show <jobID>``.
 - Compliance exports no longer time out when uploading to S3.
 - Users can now supply a certificate authority (CA) file and client certificates for the Elasticsearch client.
 - Grafana metrics are now available for database connection metrics. They are:
    - ``max_open_connections``
    - ``open_connections``
    - ``in_use_connections``
    - ``idle_connections``
    - ``wait_count_total``
    - ``wait_duration_seconds_total``
    - ``max_idle_closed_total``
    - ``max_idle_time_closed_total``
    - ``max_lifetime_closed_total``
 - Boards are served as an in-built product from within Mattermost server instead of a plugin and is now always enabled. While running in product mode, the Boards plugin will remain disabled. 

>  - Added a new section in the **System Console** for products. For now, it only contains Boards-specific settings.

Reviewer 3: Will this be available for cloud admins as well? @agnivade

Reviewer: I believe this is a self-hosted feature only.

Reviewer 2: This should be available for cloud admins as well.

Reviewer: Ah, I can see why too. It's a product vertical config option for Boards. Thanks, Agniva.

 - Users can now monitor the progress of the bulk export job via its metadata field. It is available at ``mmctl export job show <jobID>``.
 - Compliance exports no longer timeout when uploading to S3.
 - Users can now supply a certificate authority (CA) file and client certificates for the Elasticsearch client.
 - Grafana metrics are now available for database connection metrics. They are:
    - ``max_open_connections``
    - ``open_connections``
    - ``in_use_connections``
    - ``idle_connections``
    - ``wait_count_total``
    - ``wait_duration_seconds_total``
    - ``max_idle_closed_total``
    - ``max_idle_time_closed_total``
    - ``max_lifetime_closed_total``
 - Boards are served as an in-built product from within Mattermost server instead of a plugin. While running in product mode, the boards plugin will remain disabled.
 - Added a new section in the **System Console** for products. For now, it only contains Boards-specific settings.
 - Boards is now delivered as an official in-product vertical instead of a plugin and is now always enabled.
 - Made ``registerChannelIntroButtonAction`` plugin API usable by plugins other than Boards.

>  - The following new HTTP headers and values are now written on all responses. These default values should make sense in most installations and can be overridden by a reverse proxy or ingress configuration. Note that the empty ``Permissions-Policy`` does not have any other effect than be present, to suggest changing it to a more restrictive value based on actual needs.

Author: Adding @oh6hay to see if this can be simplified.

>  - The following new HTTP headers and values are now written on all responses. These default values should make sense in most installations and can be overridden by a reverse proxy or ingress configuration. Note that the empty ``Permissions-Policy`` does not have any other effect than be present, to suggest changing it to a more restrictive value based on actual needs.

Reviewer 4: How about

> Note that the empty ``Permissions-Policy`` header does not have any actual effect. Users are recommended to change it to a more restrictive value based on their use case. For more information, see the [W3C Reference](https://www.w3.org/TR/permissions-policy/) or [this article](https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy).

>  - The following new HTTP headers and values are now written on all responses. These default values should make sense in most installations and can be overridden by a reverse proxy or ingress configuration. Note that the empty ``Permissions-Policy`` does not have any other effect than be present, to suggest changing it to a more restrictive value based on actual needs.

Reviewer: Thanks, @oh6hay! Much clearer!

	```
	Permissions-Policy: 
	Referrer-Policy: no-referrer
	X-Content-Type-Options: nosniff
	```

### Bug Fixes
 - Fixed an issue with the plugin ``/public`` handling for subpaths.
 - Fixed an issue where selecting **Pinned** on a post in the Threads view would result in the right-hand side being stuck in a loading state.
 - Fixed an issue where the profile popover did not dismiss when opening a modal through a shortcut.

>  - Fixed an issue where the "Run Deletion Job Now" button for Data Retention wasn’t disabled when all policies were set to "keep forever".

Reviewer: ```suggestion
 - Fixed an issue where the **Run Deletion Job Now** button for Data Retention wasn’t disabled when all policies were set to **keep forever**.
```
